layouts/shortcodes/bildecredits-sv.html

```diff
@@ -0,0 +1,14 @@
+<p>
```

Comment:
@Stigjb Først tenkte jeg å legge til informasjon om medielisenser per trekkspill, men det ble for mye i hver. Så jeg skilte det ut og tenkte å legge det bak en annen type kollapserende knapp enn trekkspillene. Jeg tok utgangspunkt i shortcode ingress.html, hvor jeg tenkte å ha "innholdet" i .md-filen. Dessverre klarte den ikke å konvertere min markdown med lenker og alt, så da har jeg gjort dette som en midlertidig workaround.